File: client/Client/Network/Messages/Game/ItemPickedUpMessage.cs
using LoadRunnerClient.DTOs;

namespace LoadRunnerClient.Network.Messages
{
    public class ItemPickedUpMessage
    {
        public const string TYPE = "ItemPickedUpMessage";

        public ItemPickedUpMessage()
        {
        }

        /// <summary>
        /// ItemPickedUpMessage Constructor
        /// </summary>
        /// <param name="uuid"></param>
        /// <param name="itemDTO"></param>
        public ItemPickedUpMessage(string uuid, ItemDTO itemDTO)
        {
            this._uuid = uuid;
            this._itemDTO = itemDTO;
        }

        private string _uuid;

        public string uuid
        {
            get => _uuid;
            set => _uuid = value;
        }

        private ItemDTO _itemDTO;

        public ItemDTO itemDTO
        {
            get => _itemDTO;
            set => _itemDTO = value;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/game/ItemPickedUpMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.game;

import de.hsrm.mi.swtpro03.server.game.dtos.ItemDTO;

public class ItemPickedUpMessage {

    private String uuid;
    private ItemDTO itemDTO;

    public ItemPickedUpMessage() {
    }

    public ItemPickedUpMessage(String uuid, ItemDTO itemDTO) {
        this.uuid = uuid;
        this.itemDTO = itemDTO;
    }

    public String getUuid() {
        return uuid;
    }

    public void setUuid(String uuid) {
        this.uuid = uuid;
    }

    public ItemDTO getItemDTO() {
        return itemDTO;
    }

    public void setItemDTO(ItemDTO itemDTO) {
        this.itemDTO = itemDTO;
    }
}
